krun.py

```diff
@@ -187,6 +187,22 @@ def main(parser):
 
     attach_log_file(config, args.resume)
 
+    mail_recipients = config.MAIL_TO
+    if type(mail_recipients) is not list:
```

Comment:
Does it have to be a list or can it be any `collections.Sequence` type except `str`?